capty.py

```
@@ -191,59 +572,375 @@ def __init__(self):
+        """Position window at bottom center of screen"""
+        try:
+            screen = Gdk.Screen.get_default()
+            if screen:
+                screen_width = screen.get_width()
+                screen_height = screen.get_height()
+                window_width = 500
+                window_height = 50
+                x = (screen_width - window_width) // 2
+                y = screen_height - window_height - 50  # 50px from bottom
+                self.window.move(x, y)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)